Skype/SDN/articles/installing-the-sdn-manager.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Installing the SDN Manager
- 
+description: Describes the setup wizard process for installing the SDN Manager component and provides the detailed steps and information for installation.
```

Comment:
Describes the setup wizard process for installing the SDN Manager component and provides the detailed steps and information.

Reply:
updated